docs/src/man/guide.md

```diff
@@ -141,8 +141,11 @@ build/
     not resolve directory URLs like `foo/` to `foo/index.html` for local files. You have two
     options:
 
-    1. You can run a local web server out of the `docs/build` directory. If you have Python
-       installed, you can simple start one with `python3 -m http.server --bind localhost`
+    1. You can run a local web server out of the `docs/build` directory. One way to accomplish
+       this is to install the [LiveServer](https://github.com/asprionj/LiveServer.jl) Julia
+       package. You can then start the server with
+       `julia -e 'using LiveServer; cd("docs/build"); serve()'`. Alternatively, if you have Python
```

Comment:
`serve(dir="docs/build")` might be nicer.

Reply:
https://github.com/JuliaDocs/Documenter.jl/pull/1385